### ISSUE TITLE
Specify files to load about pages from

### ISSUE THREAD

**Issue by marrus-sh** [glitch-soc/mastodon#251]
Right now about page information is placed inside of a text box in the admin panel. It'd be nice if there was an option for loading from a file instead, which would allow the information to be versioned more easily.

This potentially goes for other `about` informations, as well.

**Comment by beatrix-bitrot** [comment]
yeah i would love this. an HTML fragment is fine i guess. markdown would be also nice maybe?

**Comment by marrus-sh** [comment]
HAML can parse markdown so we should be able to do both I would think~
